pom.xml

```diff
@@ -38,6 +38,8 @@
         <changelist>-SNAPSHOT</changelist>
         <jenkins.version>2.332.4</jenkins.version>
         <gitHubRepo>jenkinsci/active-choices-plugin</gitHubRepo>
+        <node.version>18.16.0</node.version>
+        <yarn.version>1.22.19</yarn.version>
```

Comment:
Where are these values used? I thought you'd use that Maven frontend-maven-plugin e.g. https://github.com/apache/jena/blob/f0476039fb3b92662554c2c11a29343261aebc02/jena-fuseki2/jena-fuseki-ui/pom.xml#L64C29-L64C50

Reply:
The jenkins plugin parent pom looks at the `.mvn_exec_yarn` marker file and adds that plugin block with `mvntest` and `mvnbuild` scripts in the `package.json`

Reply:
#TIL! Thanks!